### ISSUE TITLE
Skinned Mesh Physics (SMP) source code for Skyrim LE

### ISSUE THREAD

**Issue by Karandra** [HydrogensaysHDT/hdt-skyrimse-mods#1]
Can you upload SMP source code for Skyrim LE?

There seems to be an issue with dot vs comma for floating point numbers. It was fixed for SE by aers (https://github.com/aers/hdt-skyrimse-mods/commit/4e23bf05c6d31642b2593b19621e41b30b7ba63a) but it's still there for LE.

**Comment by MrGrymReaper** [comment]
@KerberX I think you mean the issue was fixed for SE by aers (aers@4e23bf0) but it's still there for LE.
@HydrogensaysHDT Do you think you could please do what you did for the SE version of HDT SMP for the LE version also?

**Comment by Karandra** [comment]
> I think you mean the issue was fixed for SE but it's still there for LE.

Yeah. I think this is why everyone told me about I put "LE" there twice.

Well, more than month has passed. I don't think we'll see any answer from HydrogensaysHDT.
